nixos/modules/services/hardware/fancontrol.nix

```diff
@@ -0,0 +1,39 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.hardware.fancontrol;
+
+in {
+
+  options.hardware.fancontrol = {
+    enable = mkOption {
```

Comment:
Please use `mkEnableOption` here. That is much shorter.

Reply:
i did think this was a bit verbose, thanks